src/test/java/it/pagopa/interop/probing/eservice/operations/unit/controller/EserviceControllerTest.java

```diff
@@ -135,7 +142,7 @@ void testSaveService_whenGivenValidEserviceSaveRequest_thenReturnsId() throws Ex
 		RequestBuilder requestBuilder = MockMvcRequestBuilders.post(String.format(saveEserviceUrl))
 				.contentType(MediaType.APPLICATION_JSON)
 				.content(new ObjectMapper().writeValueAsString(eserviceSaveRequest));
-		Mockito.when(service.saveEservice(eserviceSaveRequest)).thenReturn(1L);
+		Mockito.when(service.saveEservice(saveEserviceDto)).thenReturn(1L);
```

Comment:
ObjectMapper should be a singleton bean with centralized configuration

Reply:
Okay, I added it